src/resources/MLModels/MLModels.ts

```diff
@@ -0,0 +1,41 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+import {CreateMLModelOptions, MLModel, MLModelInfo} from './MLModelsInterfaces';
+
+export default class MLModels extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning/models`;
+    static createUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning/model`;
+    static infoUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning/engines`;
+
+    list() {
+        return this.api.get<MLModel[]>(MLModels.baseUrl);
+    }
+
+    listDetails() {
+        return this.api.get<MLModel[]>(`${MLModels.baseUrl}/details`);
+    }
+
+    getWithId(modelId: string) {
+        return this.api.get<MLModelInfo>(`${MLModels.baseUrl}/${modelId}/details`);
+    }

```

Comment:
It might be easier to read 'getDetailsWithId' or 'getDetailsById'

Reply:
The file MLModels.ts has already been removed. I refactored the folder structure, and now Models and ModelInformation are separated. Please refresh this page to get the latest version 😃